pkg/controllers/user/pipeline/engine/jenkins/mappper.go

```diff
@@ -0,0 +1,182 @@
+package jenkins
```

Comment:
too much code in jenkins/hooks/remote pkg for me to review : (. I assume this is copied from 1.6 so it is good already